config/regex-url-extractors.txt

```diff
@@ -3,3 +3,5 @@
 # 3. append to url before extraction
 (?<url>https:\/\/twitter.com\/[a-zA-Z0-9_]+) [data-testid="UserDescription"]
 (?<url>https:\/\/www.youtube.com\/(c|channel)\/[a-zA-Z0-9_-]+) #description /about
+(?<url>https:\/\/superhero.com\/tip\/\d+(?:_v\d+)?) .tip__body .address
```

Comment:
the goal is to add `.address` to the url?

Reply:
no its supposed to be part of the CSS selector

Reply:
this doesn't work with these selectors as currently implemented, but I can work around this